ospf6d/ospf6_message.c

```diff
@@ -2248,6 +2248,17 @@ void ospf6_hello_send(struct event *thread)
 	if (oi->gr.hello_delay.t_grace_send)
 		return;
 
+	/* Check if config is still being processed */
+	if (event_is_scheduled(t_ospf6_cfg)) {
```

Comment:
Shouldn't this be reversed (if the config is in a reading phase, let's not process hello messages)?

Reply:
No - if we're processing the configuration file, we don't want to send hellos.

Reply:
Am I misreading this or is this like: if the configuration is loading, start sending (using ospf6_hello_send()) hello messages after a specified number of seconds pass (10 default)? I mean, I don't understand why we need to schedule a thread here :shrug: (I might be absolutely wrong)

Reply:
Ah, sorry, didn't see that it's the same function, lol :)